arch/arm/soc/atmel_sam/sam4s/Kconfig.soc

```diff
@@ -112,7 +105,15 @@ config SOC_ATMEL_SAM4S_PLLB_DIVB
 	Board config file can override this settings
 	for a particular board.
 
-	With default of MULB == 6, and DIVB == 1,
-	PLL is running at 7 times of main clock.
+	With default of MULB == 3, and DIVB == 1,
+	PLL is running at 4 times of main clock.
+
+config SOC_ATMEL_SAM4S_EFC0_FMR_FWS
```

Comment:
FWS value can be either calculated statically from SYS_CLOCK_HW_CYCLES_PER_SEC or dynamically at run time in atmel_sam4s_init() function. We should not add a new config option.